src/java/org/apache/cassandra/tools/NodeProbe.java

```diff
@@ -1717,6 +1718,16 @@ public void resumeBootstrap(PrintStream out) throws IOException
         }
     }
 
+    public Map<String, List<Integer>> getMaximumPoolSizes(List<String> filter)
```

Comment:
Is this a filter, or a set of names to select?

Reply:
More accurately a set of names to select - I was thinking of it filtering the list of available stage names, but an incorrect name will trigger a message letting the user know the canonical names for the stages.  I've renamed to `stageNames` instead. 

```
Jons-PIE-MBP:trunk-work jmeredith$ bin/nodetool getconcurrency
Pool Name                    CorePoolSize MaximumPoolSize
AntiEntropyStage                        1               1
CounterMutationStage                    0              32
GossipStage                             1               1
ImmediateStage                          0               0
InternalResponseStage                  12              12
MigrationStage                          1               1
MiscStage                               1               1
MutationStage                           0              32
ReadStage                               0              32
RequestResponseStage                    0              12
TracingStage                            1               1
ViewMutationStage                       0              32
Jons-PIE-MBP:trunk-work jmeredith$ bin/nodetool getconcurrency x
Pool Name                    CorePoolSize MaximumPoolSize
nodetool: Must be one of READ,MUTATION,COUNTER_MUTATION,VIEW_MUTATION,GOSSIP,REQUEST_RESPONSE,ANTI_ENTROPY,MIGRATION,MISC,TRACING,INTERNAL_RESPONSE,IMMEDIATE
See 'nodetool help' or 'nodetool help <command>'.
```